scripts/generate-webplugin-chrome-dev.sh

```diff
@@ -0,0 +1,34 @@
+# Build the webplugin
```

Comment:
This script and the other builds scripts show a __lot__ of similarities. Please consider rewriting them so a single build file is used. Minor platform differences (like Edge requiring extra plugin files) could be handled within the script. If Safari is very different we may consider using a separate script for Safari.

Bash is quite flexible :) I'm sure we can do a lot in a single script!